### PR TITLE
Give 60 day internal warning of subscription end

### PR DESCRIPTION
**Pull request by esoergel**
http://manage.dimagi.com/default.asp?232729
@czue
@proteusvacuum FYI

### CODE REVIEW

File: corehq/apps/accounting/tasks.py [hunk 1]
```diff
@@ -286,7 +286,7 @@ def remind_dimagi_contact_subscription_ending_40_days():
     """
     Sends reminder emails to Dimagi contacts that subscriptions are ending in 40 days
```
Comment (proteusvacuum) [inline corehq/apps/accounting/tasks.py]: the docstring and function names are no longer correct

Comment (esoergel) [inline corehq/apps/accounting/tasks.py]: good call.  Updated.

**Comment by proteusvacuum** [comment]
:+1: on :white_check_mark:

**Comment by esoergel** [comment]
:white_check_mark:

**Pull request merged**